microsoft-edge/webview2/releasenotes.md

```diff
@@ -47,7 +51,10 @@ Release Date: January 25, 2021
 ###### Promotions  
 *   Promoted the following experimental changes to stable:
     * Visual Hosting APIs.
-    * URI Protocol Scheme.
+    * [SetVirtualHostNameToFolderMapping][ReferenceWin32Icorewebview210774PreReleaseSetVirtualHostNameToFolderMapping]
+    * [TrySuspend & Resume][ReferenceWin32Icorewebview210774PreReleaseTrySuspendResume]
```

Comment:
There are no experimental versions of TrySuspend & Resume and DefaultBackgroundColor. They are fast-tracked to public IDL directly.

Reply:
Yes, but they are stable in the prerelease, so I noted it as a 'promotion', even though it wasn't experimental.  Will add a note.